src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -335,10 +338,56 @@ def mark_node(self, node):
             node.set_friendly_name(name)
         return node
 
+    @staticmethod
+    def convert_quantized_tensor(qtensor: torch.Tensor):
```

Comment:
Are those modifications used be quantize/dequantize? From what I see this is needed for quantized linear and convs, maybe it would be best to remove those changes in decoder and getattr from this PR and add it with those ops so this could be tested? Also convert_quantized_tensor would likely need to be modified to use transformation for QuantizedNode

Reply:
Right, removed!